common/src/main/java/io/druid/concurrent/TaskPriority.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Licensed to Metamarkets Group Inc. (Metamarkets) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  Metamarkets licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.druid.concurrent;
+
+public class TaskPriority
+{
+  // The task context key to grab the task priority from
+  public static final String CONTEXT_KEY = "taskPriority";
```

Comment:
With the changes in https://github.com/druid-io/druid/pull/1679, 
there will be two context values "priority" and "taskPriority" which looks similar but have different semantics. 
Can we rename this to taskThreadPriority to be more explicit ?